test-app/tests/integration/components/headless-form-test.gts

```diff
@@ -393,19 +558,26 @@ module('Integration Component headless-form', function (hooks) {
 
         await fillIn('input[data-test-first-name]', 'Nicole');
         await fillIn('input[data-test-last-name]', 'Chung');
+        await click('input[data-test-gender-female]');
         await click('input[data-test-terms]');
         await click('[data-test-submit]');
 
         assert.deepEqual(
           data,
-          { firstName: 'Tony', lastName: 'Ward', acceptTerms: false },
+          {
+            firstName: 'Tony',
+            lastName: 'Ward',
+            gender: 'male',
+            acceptTerms: false,
+          },
           'data is not mutated'
         );
 
         assert.true(
           submitHandler.calledWith({
             firstName: 'Nicole',
             lastName: 'Chung',
+            gender: 'female',
```

Comment:
Question: do we need to have an assert explicitly for `FormData` or does the assert on the `submitHandler` effectively check for this?

Reply:
Sorry, not sure what you mean here?